checks.d/linuxvm.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+"""DataDog agent checks for system.vm proc info."""
+
+from checks import AgentCheck
+
+
+def valid_instance(instance):
+    return "proc" in instance or "vmstat" in instance
+
+
+class LinuxVm(AgentCheck):
+    VM_BASE = "system.vm"
+
+    def check(self, instance):
+        if not valid_instance(instance):
+            return
+
+        proc = instance.get("proc", {})
```

Comment:
you don't need to set default values as you checked before that these keys were in the instance dict.